### PR TITLE
row refresh fix

### PR DESCRIPTION
**Pull request by prv-proton**

### CODE REVIEW

File: frontend/src/views/FuelCodes/AddFuelCode/AddFuelCode.jsx [hunk 1]
```diff
@@ -127,7 +128,37 @@ export const AddFuelCode = () => {
...
     })
-    validationHandler(params)
+    const focusedCell = params.api.getFocusedCell()
+    if (focusedCell.column.colId === 'fuelCode') {
+      const fuelCodeData = optionsData.latestFuelCodes.find(
+        (fuelCode) => fuelCode.fuelCode === params.data.fuelCode
+      )
+      params.node?.setData({
+        ...params.data,
+        prefix: 'BCLCF',
```
Comment (prv-proton) [inline frontend/src/views/FuelCodes/AddFuelCode/AddFuelCode.jsx:138]: hardcoded value, please change as requrired.

File: frontend/src/views/FuelCodes/AddFuelCode/_schema.jsx [hunk 2]
```diff
@@ -186,22 +145,28 @@ export const fuelCodeColDefs = (t, optionsData) => [
...
     cellEditor: 'autocompleteEditor',
-    editable: true,
     cellEditorParams: {
-      onDynamicUpdate: (fuelCodeValue, params) =>
-        autoFill(
-          { fuelCodeValue, latestFuelCodes: optionsData.latestFuelCodes },
-          params
-        ),
-      autoSelect: true,
+      onDynamicUpdate: (params) => params.api.redrawRows({ rowNodes: [params.node] }),
```
Comment (prv-proton) [inline frontend/src/views/FuelCodes/AddFuelCode/_schema.jsx]: autoFill method was updating only the cells that aren't visible within the grid view, hence had to use redraw data row and using onRowEditingStopped handler to update the row.

File: frontend/src/views/FuelCodes/AddFuelCode/AddFuelCode.jsx [hunk 3]
```diff
@@ -136,6 +167,15 @@ export const AddFuelCode = () => {
     // Add your API call to save modified rows here
   }, [])
 
+  useEffect(() => {
```
Comment (prv-proton) [inline frontend/src/views/FuelCodes/AddFuelCode/AddFuelCode.jsx:170]: this ensures the row re-enters into editing mode after row is updated.

**Review by kevin-hashimoto — APPROVED** [review]
Works great, thanks for your help!

**Pull request merged**